index.js

```diff
@@ -1,17 +1,56 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items = this.items.sort((a,b) => a - b);
+    this.length++;
+    // return item.sort(); 
+    // return sorted;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos === null || pos > this.items.length) {
+      throw TypeError("OutOfBounds");
+    } else {
+      return this.items.indexOf(pos);
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.length === 0) {
+      throw TypeError("EmptySortedList");
+    } else {
+      return this.items.slice(-1).pop()
+    }
+  }
 
-  min() {}
+  min() {
+    if (this.length === 0) {
+      throw TypeError("EmptySortedList");
+    } else {
+      return this.items[0];
```

Comment:
Well, the number in the index 0, can not be the smallest, has the example before, try use **Math.min()**

You're wellcome 👍